invenio_users_resources/administration/views/users.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio administration users view module."""
+from invenio_administration.views.base import (
+    AdminResourceCreateView,
+    AdminResourceDetailView,
+    AdminResourceEditView,
+    AdminResourceListView,
+)
+from invenio_i18n import lazy_gettext as _
+
+
+class UsersListView(AdminResourceListView):
+    """Configuration for users sets list view."""
+
+    api_endpoint = "/users/sets"
+    name = "USERS"
+    resource_config = "users_resource"
+    search_request_headers = {"Accept": "application/json"}
+    title = "USERS Sets"
```

Comment:
_Sets_ is a specific term used on OAI-PMH

```suggestion
    title = "Users"
```